src/backend/utils/storage.js

```diff
@@ -29,11 +27,13 @@ module.exports = {
   getFeedsCount: () => redis.scard(FEEDS),
 
   addPost: async post => {
+    const encrGuid = standardize(post.guid, 'post');
```

Comment:
This isn't encrypted, so maybe call it `hashedGuid` or just `key` to better communicate its use.

Reply:
Renamed `encrGuid` to `key`.